src/Microsoft.DotNet.Wpf/src/ReachFramework/Packaging/XpsFixedDocumentReaderWriter.cs

```diff
@@ -1,1278 +1,1282 @@
-        // and current child of this node for walking up and
-        // down the current tree.  This is used be the flushing
-        // policy to do interleave flushing of parts correctly.
-        //
-        private INode                               _parentNode;
-
-        //
-        // This variable flags wehter the pageCashe
-        // has been populated by parsing the part for dependent pages
-        //
-        private bool                                _hasParsedPages;
-
-        //
-        // 0 based document number in the document sequence
-        //
-        private int                                 _documentNumber;
-
-        private XpsThumbnail                        _thumbnail;
-
-
-        //
-        // Since the current page may add link target information
-        // we must track our current page
-        // for this reason we can not handle adding new page
-        // until the the current page has been committed
-        //
-        private XpsFixedPageReaderWriter            _currentPage;
-        //
-        // A cached list of Signature Definitions
-        //
-        private Collection<XpsSignatureDefinition>  _signatureDefinitions;
-        //
-        // Boolean indicating whetehr _signatureDefinitions collection
-        // has been changed
-        //
-        private bool                                _sigCollectionDirty;
-        
-        private XpsStructure                        _documentStructure;
-        #endregion Private data
-
-        #region Internal properties
-
-        /// <summary>
-        /// Gets a reference to the XmlWriter for the Metro part
-        /// that represents this fixed document.
-        /// </summary>
-        internal System.Xml.XmlWriter XmlWriter
-        {
-            get
-            {
-                _partEditor.DoesWriteStartEndTags = false;
-                return _partEditor.XmlWriter;
-            }
-        }
-
-        #endregion Internal properties
-
-        #region INode implementation
-
-        void
-        INode.Flush(
-            )
-        {
-            //
-            // Commit the PrintTicket (if necessary)
-            //
-            CommitPrintTicket();
-
-            //
-            // Create the relationship between the package and the tumbnail
-            //
-            if( _thumbnail != null )
-            {
-                _thumbnail = null;
-            }
-        }
-
-        void
-        INode.CommitInternal(
-            )
-        {
-            CommitInternal();
-        }
-
-        PackagePart
-        INode.GetPart(
-            )
-        {
-            return _metroPart;
-        }
-
-        #endregion INode implementation
-
-        #region IDisposable implementation
-
-        void
-        IDisposable.Dispose(
-            )
-        {
-            CommitInternal();
-        }
-
-        #endregion IDisposable implementation
-    }
-}
-
+// Licensed to the .NET Foundation under one or more agreements.

```

Comment:
weird, viewing the diff in VS looks normal, not sure why this looks like the whole doc changed? maybe line-endings?

Reply:
What's the actual change to look at?

Reply:
it's just wrapping a call to `PackagePart.GetStream` in a using statement

Here are some pics of the diff in VS that lets you see the changes properly:
![image](https://user-images.githubusercontent.com/9649518/61477449-1f544f00-a944-11e9-899f-689387e3631b.png)

![image](https://user-images.githubusercontent.com/9649518/61477463-2d09d480-a944-11e9-9e83-bb4454b69b0b.png)